src/data/uma-nickname-requirements.csv

```diff
@@ -30,18 +30,18 @@
 "芝のレースで15勝する","Win a turf race 15 times"
 "20回レースに出走する","Participate a race 20 times"
 "ダートのレースで10勝する","Win a dirt race 10 times"
-"オープン以下で負けずに、重賞を5連勝する","Win a big race 5 times in a row while having zero losses in a grade higher than OP"
+"オープン以下で負けずに、重賞を5連勝する","Win a graded race 5 times in a row while having zero losses in a grade higher than OP"
 "シニア級で初めてGⅠを勝利する","Win a GI race for the first time in 3rd year"
 "GⅠで3着を3回とる","Finish 3rd place in a GI race 3 times"
 "GⅠで2着を3回とる","Finish 2rd place in a GI race 3 times"
 "スキルを15個以上獲得する","Learn ≥15 skills"
 "基礎能力[スピード]が800以上かつ\n芝のレースを15回以上出走する","Participate a turf race ≥15 times\nReach ≥800 speed"
-"重賞未勝利でGⅠを勝利する","Win a GI race while still haven't won any big race"
+"重賞未勝利でGⅠを勝利する","Win a GI race while still haven't won any graded race"
```

Comment:
Not grammatically correct, maybe "Win a GI race without winning any graded races before that"

Reply:
That's true. Wasn't paying much attention to the rest, just ctrl+f'd all instances of big and changed where needed.

Reply:
I've changed it to "Win a GI race without having won any other graded races"